.env

```diff
@@ -1,8 +1,8 @@
-VIRTUAL_HOST=localhost3df
+VIRTUAL_HOST=localhost
 USE_TLS=false
 
-# KEY_FILE_PATH=
+# KEY_FILE_PATH={path to key file, used when "USE_TLS" is set to true}
 # CERT_FILE_PATH={path to certificate file, used when "USE_TLS" is set to true}
-# STUN_SERVERS=
-# TURN_SERVERS=
-# EXTERNAL_IP=
+# STUN_SERVERS={list of IPv4 addresses of the STUN servers}
+# TURN_SERVERS={list of IPv4 addresses of the TURN servers}
```

Comment:
These 2 environment variables are not in use any longer

Reply:
I have removed them